src/main/java/org/sonarsource/auth/github/GitHubSettings.java

```diff
@@ -38,6 +38,7 @@
 
   //TODO remove TODO
   private static final String CLIENT_ID = "sonar.auth.github.clientId.secured";
+  //TODO Another todo, a casual one
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.auth.github%3Asonar-auth-github-plugin&pullRequest=28&issues=AWJKsgawuc1UcxX5-Tas&open=AWJKsgawuc1UcxX5-Tas)

[ISSUE_KEY]: AWJKsgawuc1UcxX5-Tas